### ISSUE TITLE
v4.1.0 RELEASE BUG: Could not create endpoint for path: /eth/v1/beacon/blinded_blocks ( REST endpoint)

### ISSUE THREAD

**Issue by james-prysm** [prysmaticlabs/prysm#13071]
### Describe the bug

discord user #pacobits from stakely first reported on discord https://discord.com/channels/476244492043812875/476588476393848832/1164235499813224549

Calling the POST request endpoint `/eth/v1/beacon/blinded_blocks` on the prysm beacon node results in the error `Could not create endpoint for path: /eth/v1/beacon/blinded_blocks` meaning the endpoint is not available in REST. 

using the prysm validator with prysm beacon node works successfully, only the REST endpoint is not functioning in v4.1.0
if you are using a LH validator with prysm or vouch you will have this issue.

**note:** the V2 version of the endpoint [/eth/v2/beacon/blinded_blocks](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/publishBlindedBlockV2) should still be available and accessible

### Has this worked before in a previous version?

```text
Yes this worked in a previous version
```

### 🔬 Minimal Reproduction

Steps to reproduce:

1. get the v4.1.0 prysm beacon node
2. sync to head
3. call the  `/eth/v1/beacon/blinded_blocks`  rest endpoint as a `POST` request
4. get the following error `Could not create endpoint for path: /eth/v1/beacon/blinded_blocks`

### Error

```text
Could not create endpoint for path: /eth/v1/beacon/blinded_blocks
```

### Platform(s)

Linux (x86), Linux (ARM), Mac (Intel), Mac (Apple Silicon), Windows (x86), Windows (ARM)

### What version of Prysm are you running? (Which release)

v4.1.0

**Comment by prestonvanloon** [comment]
Likely root cause: https://github.com/prysmaticlabs/prysm/pull/12827

**Comment by prestonvanloon** [comment]
Probably fixed as of https://github.com/prysmaticlabs/prysm/pull/13048

**Comment by prestonvanloon** [comment]
Cherry-picked a7e56058c18a06ed3fce670bec2426420893093d f592bf7f075ce0171fd2ae1ac84d8b680c52d058 f91efafe24409e25180d0aeb6488e59c4b9f1b79 into [v4.1.1 release branch](https://github.com/prysmaticlabs/prysm/compare/develop...release-v4.1.1)

**Comment by prestonvanloon** [comment]
Fixed with [v4.1.1](https://github.com/prysmaticlabs/prysm/releases/tag/v4.1.1)
